unused_test.go

```diff
@@ -131,22 +131,22 @@ func TestUnused(t *testing.T) {
 		{
 			name: "must consume Unused",
 			chain: []any{
-				MustConsume(Required(func() Unused { called = true; return Unused{} })),
+				MustConsume[Unused](Required(func() Unused { called = true; return Unused{} })),
```

Comment:
Don't `MustConsume` and `Unused` signal conflicting intentions? `Unused` should not be used, right?

Reply:
MustConsume and Unused do conflict. I've decided that Unused trumps MustConsume. I'm not going to bother making it an error to combine them though.